CommunityToolShedMvc/Models/Tool.cs

```diff
@@ -8,11 +9,14 @@ namespace CommunityToolShedMvc.Models
     public class Tool
     {
         public int Id { get; set; }
+
+        [Display(Name = "Item Name")]
         public string ItemName { get; set; }
-        public int OwnerId { get; set; }
+
+        public int OwnerId { get; internal set; }
```

Comment:
Using the `internal` access modifier here doesn't have any noticeable effect as all of your application code resides in a single project. If you defined this class in its class library project, then using the `internal` access modifier would restrict access to the property setter to classes within that project.